### PR TITLE
[SPARK-24898][DOC] Adding spark.checkpoint.compress to the docs

### PR DESCRIPTION
**Pull request by sandeepvja**
## What changes were proposed in this pull request?

Adding spark.checkpoint.compress configuration parameter to the documentation

## How was this patch tested?

Checked locally for jeykyll html docs. Also validated the html for any issues.

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -1107,6 +1107,14 @@ Apart from these, the following properties are also available, and may be useful
     Compression will use <code>spark.io.compression.codec</code>.
   </td>
 </tr>
+<tr>
+  <td><code>spark.checkpoint.compress</code></td>
+  <td>false</td>
+  <td>
+    Whether to compress RDD checkpoints. Generally a good idea.
+    Compression will use <code>spark.io.compression.codec</code>
```
Comment (dongjoon-hyun) [inline docs/configuration.md]: Please add a `.` at the end of the sentence.

**Comment by dongjoon-hyun** [comment]
Thank you for your first contribution, @sandeepvja . I left a minor comment.

**Comment by dongjoon-hyun** [comment]
ok to test

**Review by HyukjinKwon — APPROVED** [review]
Looks good except the comments above.

File: docs/configuration.md (hunk 1 above)
Comment (sandeepvja) [reply docs/configuration.md]: @dongjoon-hyun added a `.` at the end of the sentence

**Comment by sandeepvja** [comment]
Screen Shot 2019-06-17 at 10 51 51 AM

**Comment by sandeepvja** [comment]
> Thank you for your first contribution, @sandeepvja .
> 
> * I left a minor comment.
> * Please attach the generated screenshot to the PR description.

@dongjoon-hyun Thanks for reviewing my first PR. Attached the screenshot to the issue.

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.
Thank you, @sandeepvja and @HyukjinKwon .
Merged to master/branch-2.4/branch-2.3.

**Comment by dongjoon-hyun** [comment]
@sandeepvja . What is your JIRA ID? You will be an Apache Spark contributor as the `Assignee` of https://issues.apache.org/jira/browse/SPARK-24898 .

**Comment by sandeepvja** [comment]
my JIRA Id: m.sandeep , Can you please add me as a contributor to the JIRA issue ?

**Comment by dongjoon-hyun** [comment]
Sure. @sandeepvja . You are added to the Apache Spark contributor group and the issue is assigned to you. Thank you, @sandeepvja !

**Pull request closed without merging**